client/coral-embed-stream/src/Embed.js

```diff
@@ -144,7 +158,10 @@ class Embed extends Component {
       <div style={expandForLogin}>
         <div className="commentStream">
           <TabBar onChange={this.changeTab} activeTab={activeTab}>
-            <Tab><Count count={asset.totalCommentCount}/></Tab>
+            <Tab><Count
+              count={asset.totalCommentCount}
+              handleClick={this.handleClick}/>
```

Comment:
I think the `refetching` logic should be in `changeTab` and not on a clickable `Count`.
There is already a logic that was wrongly changed to only `refetch` for admins. 
You should remove that check and add your logic to show all comments.